pkg/interactive/termhandler.go

```diff
@@ -253,19 +252,35 @@ func (i *interactive) printQueue() {
 	}
 }
 
-func (i *interactive) deleteQueue(in string) {
-	index, err := strconv.Atoi(in)
-	if err != nil {
-		i.Job.Output.Warning(fmt.Sprintf("Not a number: %s", in))
-	} else {
-		if index < 0 || index > len(i.Job.QueuedJobs())-1 {
-			i.Job.Output.Warning("No such queued job. Use \"queueshow\" to list the jobs in queue")
-		} else if index == 0 {
-			i.Job.Output.Warning("Cannot delete the currently running job. Use \"queueskip\" to advance to the next one")
-		} else {
-			i.Job.DeleteQueueItem(index)
-			i.Job.Output.Info("Job successfully deleted!")
+func (i *interactive) deleteQueue(in []string) {
+	sort.Slice(in, func(k, l int) bool {
+		ink, err := strconv.Atoi(in[k])
+		if err != nil {
+			// Do not log, it will be logged later
+			ink = -1
+		}
+		inl, err := strconv.Atoi(in[l])
+		if err != nil {
+			// Do not log, it will be logged later
+			inl = -1
 		}
+		return ink > inl
+	})
```

Comment:
For instance: 

```
> qd 111 a
[WARN] No such queued job. Use "queueshow" to list the jobs in queue
[WARN] Not a number: a
```

I still have to improve the "no such queued job"

Reply:
It currently just shows the warning, but will delete other entries in the command. In case your example above would have had a valid queue entry, that valid entry would have been deleted.  

While there would be a "Job XX successfully deleted!" - output, the thing that the user typically concentrates is the warning message.

Reply:
Doing a more realistic test : 
```
Queued jobs:
 [0] : https://[REDACTED]/FUZZ (active job)
 [1] : https://[REDACTED]/2/FUZZ
 [2] : https://[REDACTED]/2016/FUZZ
 [3] : https://[REDACTED]/2017/FUZZ
 [4] : https://[REDACTED]/2019/FUZZ
 [5] : https://[REDACTED]/2018/FUZZ
> qd 2 a 3
[INFO] Job successfully deleted!

[INFO] Job successfully deleted!

[WARN] Not a number: a
> q
Queued jobs:
 [0] : https://[REDACTED]/FUZZ (active job)
 [1] : https://[REDACTED]/2/FUZZ
 [2] : https://[REDACTED]/2019/FUZZ
 [3] : https://[REDACTED]/2018/FUZZ

Reply:
Note that "Job successfully deleted" is now "Job x successfully deleted"